lib/flask_tasks.py

```diff
@@ -84,13 +95,18 @@ def task(app, environ):
                 try:
                     # Run the route function and record the response
                     logging.debug("Executing target function")
+                    tasks[task_id]['thread_name'] = threading.Thread.getName(
+                        threading.current_thread())
+                    tasks[task_id]['log_file'] = threaded_logging. \
+                        get_log_name(tasks[task_id]['thread_name'])
```

Comment:
Why won't you move the whole **task** method outside of async_task, so you do not need this line length fighting...

Reply:
I'm don't really understand what do you mean. Quoted code is a self-identification of thread for separating from other threads.  Threads are generated via wrapping flask requests, so time to be ready for next request is small as possible.

Reply:
okay, I give more explanatory example:
instead of doing that:
```
def async_task(f):
    @wraps(f)
    def wrapped(*args, **kwargs):
         def task(app, environ):
                .
                .
                .
```
do this:
```
def task(app, environ):
    .
    .
    .

def async_task(f):
    @wraps(f)
    def wrapped(*args, **kwargs):
            .
            .
            .
```

Also I don't see where the task is used...

Reply:
tasks is used ~25 lines below

```
 threading.Thread(
            target=task, args=(current_app._get_current_object(),
                               request.environ))}
```

Reply:
if code is not so ugly I prefer to keep function inside, it simplifies work *args, **kwargs parameters